### PR TITLE
Feat : Added Authentication to Pomodora app

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
Added login, logout and signup feature with guest login button

### CODE REVIEW

**Review by jalajcodes — APPROVED** [review]
Looks good!

File: src/component/pomodora.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        </div>
+      </div>
+      <div className="note-detail-container card">
+        <div className="note-container">
+          <div className="note-header">
+            <h4>Note Title...</h4>
+          </div>
+          <div className="note-body">
+            <p className="bold p2 text-grey">
+              Lorem Ipsum es simplemente el texto de relleno de las imprentas y
```
Comment (Megha-Pathak) [inline src/component/pomodora.js:28]: Suggestion: Can keep dummy text in separate file, like headerContent.js, to keep logic and content separate and easily manageble

File: src/utils/constantData/authConstant.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  password: ""
+};
+export const initialSignUpData = {
+  firstName: "",
+  lastName: "",
+  email: "",
+  password: ""
+};
+export const guestData = {
+  email: "adarshbalika@gmail.com",
```
Comment (Megha-Pathak) [inline src/utils/constantData/authConstant.js:12]: add some other user in guestData instead of adarshbalika credentials

**Review by Megha-Pathak — APPROVED** [review]

**Review by JantuDeb — APPROVED** [review]
Codes are clean and structure

File: src/App.js [hunk 3]
```diff
@@ -1,43 +1,42 @@
...
+import Navigation from "./component/navigation";
+import Home from "./component/home";
+import Archive from "./component/archive";
+import Trash from "./component/trash";
+import AddNote from "./component/addnote";
+import Pomodora from "./component/pomodora";
+import Edit from "./component/edit";
+import Login from "./component/login";
+import Signup from "./component/signup";
+import { useAuth } from "./context";
```
Comment (HeyNitin) [inline src/App.js:13]: You can use absolute imports instead of relative from next project

File: src/component/archive.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+                        })}
+                      </ul>
+                      <div className="flex flex-align-end">
+                        <p className=" p1 bold">{note.timer} Min</p>
+                      </div>
+                    </div>
+                  </div>
+                  <div className="card__actions">
+                    <p className="text-grey p1">Created on {note.date}</p>
+                    <div className="card__action-icons p3 text-grey">
```
Comment (HeyNitin) [inline src/component/archive.js:75]: why are these different standards for class names? You can try to keep the code consistent

File: src/backend/db/users.js [hunk 5]
```diff
@@ -13,6 +13,6 @@ export const users = [
     email: "adarshbalika@gmail.com",
     password: "adarshBalika123",
     createdAt: formatDate(),
```
Comment (HeyNitin) [inline src/backend/db/users.js:15]: You can change this default login credentials

File: src/component/pomodora.js (hunk 1 above)
Comment (HeyNitin) [inline src/component/pomodora.js:28]: Content can be shifted to another file and then be imported here

**Review by HeyNitin — APPROVED** [review]
Left some comments, mainly about coding practices, the rest all look good. UI is slick, great job there

**Pull request merged**